Gemfile.lock

```diff
@@ -0,0 +1,53 @@
+GEM
```

Comment:
is this file required for proper rendering of the web site? if not, I suggest it not be added.